mmv1/third_party/terraform/utils/utils.go

```diff
@@ -72,6 +72,19 @@ func getBillingProject(d TerraformResourceData, config *Config) (string, error)
 	return getBillingProjectFromSchema("billing_project", d, config)
 }
 
+func getQuotaProjectOrEmpty(d TerraformResourceData, config *Config, resourceProject string) string {
+	if !config.UserProjectOverride {
+		return ""
+	}
+	if resourceBillingProject, err := getBillingProject(d, config); err != nil {
```

Comment:
`getBillingProject` checks the resource for a `billing_project` field, and then falls back to `config.BillingProject` if there's none (which it will do in almost all cases). If there's no `billing_project` value on the resource or config it'll return an error, otherwise it will return the one it found.

You can omit the block @ L82, as that's redundant with the billing project found here.